sql/resource_sql_global_config.go

```diff
@@ -28,7 +29,7 @@ type GlobalConfig struct {
 	DataAccessConfig        map[string]string `json:"data_access_config,omitempty"`
 	InstanceProfileARN      string            `json:"instance_profile_arn,omitempty"`
 	GoogleServiceAccount    string            `json:"google_service_account,omitempty"`
-	EnableServerlessCompute bool              `json:"enable_serverless_compute,omitempty" tf:"default:false"`
+	EnableServerlessCompute bool              `json:"enable_serverless_compute,omitempty" tf:"computed,optional"`
```

Comment:
`optional` isn't required here. It's used only when we don't have `omitempty` on `json` tag...